### PR TITLE
Fix typo in api.rst

### PR DESCRIPTION
**Pull request by alexwlchan**

### CODE REVIEW

File: docs/source/api.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@ use the :class:`Priority <hyperframe.frame.Priority>` and
 attributes defined on *those* mixins available as well.
 
 Rather than clutter up the documentation repeatedly documenting those methods
-and objects, we explicitly show the inheritance heirarchy of the frames: don't
+and objects, we explicitly show the inheritance hierarchy of the frames: don't
 forget to consule the parent classes before deciding if a method or attribute
```
Comment (Lukasa) [inline docs/source/api.rst]: Want to fix the fact that 'consule' is not a word as well, while we're here?

**Comment by Lukasa** [comment]
Thanks for this! Thought of an extra fix while we're here...

**Comment by alexwlchan** [comment]
Second typo fixed.

**Comment by Lukasa** [comment]
Thanks! :sparkles:

**Pull request merged**